services/provider/server/server.go

```diff
@@ -642,17 +643,20 @@ func (s *OCSProviderServer) GetStorageClassClaimConfig(ctx context.Context, req
 			if err != nil {
 				return nil, status.Error(codes.Internal, err.Error())
 			}
-
+			data := map[string]string{
+				"clusterID": getSubVolumeGroupClusterID(subVolumeGroup),
+				"fsName":    subVolumeGroup.Spec.FilesystemName,
+				"csi.storage.k8s.io/provisioner-secret-name":       provisionerCephClientSecret,
+				"csi.storage.k8s.io/node-stage-secret-name":        nodeCephClientSecret,
+				"csi.storage.k8s.io/controller-expand-secret-name": provisionerCephClientSecret,
+			}
+			if storageClassClaim.Status.StorageProfile != "" {
+				data["pool"] = subVolumeGroup.GetLabels()[v1alpha1.CephFileSystemDataPoolLabel]
+			}
```

Comment:
the default profile will be added to status, and AFAIK it will work. if we change it may cause issues for the existing storageclasses(upgraded clusters)